pkg/token/token_meta.json

```diff
@@ -583,6 +583,18 @@
       "logo": "https://static.alchemyapi.io/images/assets/2396.png"
     }
   },
+  "XPRT": {
+    "address": "0xae837EacBAE2a6bA166ce0DEd5C72340f212835c",
+    "coinGeckoId": "persistence",
+    "denom": "ibc/a0cc0cf735bfb30e730c70019d4218a1244ff383503ff7579c9201ab93ca9293",
+    "metaSource": "custom",
+    "meta": {
+      "name": "Persistence",
+      "symbol": "XPRT",
+      "decimals": 6,
+      "logo": "https://assets.coingecko.com/coins/images/14582/large/512_Light.png?1617149658"
+    }
+  },
```

Comment:
Please double check at: https://coinmarketcap.com/currencies/persistence/
For logo: https://k8s.mainnet.asset.injective.network/asset-price/v1/coin/price?coinIds=persistence&currency=usd